src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -647,4 +651,13 @@ public ConsoleLogsPageState ConvertViewModelToSerializable()
     {
         return new ConsoleLogsPageState(PageViewModel.SelectedResource?.Name);
     }
+
+    protected override Dictionary<string, AspireTelemetryProperty> GetTelemetryProperties()
+    {
+        return new Dictionary<string, AspireTelemetryProperty>
```

Comment:
Is a dictionary necessary? A lighter weight option would be `List<KeyValuePair<string, AspireTelemetryProperty>>`. Or you could add the name to `AspireTelemetryProperty`.

Reply:
We have to send a dictionary to the aspire server, so we'll need to be creating a dictionary at some point